### PR TITLE
[SYCL][FE][Driver] Disable fp-accuracy tests

### PR DESCRIPTION
**Pull request by dm-vodopyanov**
They fail in post commit for a couple of days.

https://github.com/intel/llvm/issues/9934

### CODE REVIEW

File: clang/test/CodeGen/fp-accuracy.c
Comment (Fznamznon) [inline clang/test/CodeGen/fp-accuracy.c:33]: Can they be disabled only for failing configuration?

Comment (dm-vodopyanov) [reply clang/test/CodeGen/fp-accuracy.c:33]: Can you suggest the configuration for linux?

Comment (Fznamznon) [reply clang/test/CodeGen/fp-accuracy.c:33]: Something like `XFAIL: system-linux`.

Comment (Fznamznon) [reply clang/test/CodeGen/fp-accuracy.c:33]: How we ended up committing failing test?

Comment (dm-vodopyanov) [reply clang/test/CodeGen/fp-accuracy.c:33]: @zahiraam investigating this. The failure is only reproducible with self-build, which is only in post-commit, so there were no failures in pre-commit on Linux.

Comment (dm-vodopyanov) [reply clang/test/CodeGen/fp-accuracy.c:33]: > XFAIL will fail if test passes. If the test can pass on some linux, In order to disable we probably should use REQUIRES or UNSUPPORTED instead of XFAIL.

Right, changed to `UNSUPPORTED`.

Comment (zahiraam) [reply clang/test/CodeGen/fp-accuracy.c:33]: The tests are failing on self-build on Linux and only on a release compiler. It passes with debug self-build compiler.

**Review by Fznamznon — APPROVED** [review]

**Comment by dm-vodopyanov** [comment]
@romanovvlad could you please merge without other code owners approval? The change is obvious and should make post-commit green.

**Pull request merged**
